Mimir.Worker/Util/StateJsonConverter.cs

```diff
@@ -14,20 +15,35 @@ public override bool CanConvert(Type objectType)
         return typeof(IState).IsAssignableFrom(objectType);
     }
 
-    public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+    public override object ReadJson(
+        JsonReader reader,
+        Type objectType,
+        object existingValue,
+        JsonSerializer serializer
+    )
     {
         throw new NotImplementedException();
     }
 
     public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
     {
-        JObject jo = JObject.FromObject(value, JsonSerializer.CreateDefault(new JsonSerializerSettings { 
-            ReferenceLoopHandling = ReferenceLoopHandling.Ignore,
-            Converters = new List<JsonConverter>() { new BigIntegerToStringConverter() },
-            Formatting = Formatting.Indented,
-            NullValueHandling = NullValueHandling.Ignore
-        }));
-        
+        JObject jo = JObject.FromObject(
+            value,
+            JsonSerializer.CreateDefault(
+                new JsonSerializerSettings
+                {
+                    ReferenceLoopHandling = ReferenceLoopHandling.Ignore,
+                    Converters = new List<JsonConverter>() { new BigIntegerToStringConverter() },
+                    Formatting = Formatting.Indented,
+                    NullValueHandling = NullValueHandling.Ignore,
+                    ContractResolver = new DefaultContractResolver()
+                    {
+                        IgnoreSerializableInterface = true
```

Comment:
This setting is related to issue #77. Can you confirm how the Material is saved?

Reply:
👍

Reply:
I made sure that when this change was normalized, it would still work properly.(https://github.com/planetarium/mimir/blob/main/Mimir/Models/Items/Item.cs#L38-L56)